### PR TITLE
[FIX] l10n_pa_localization: Repair lint error sql-injection

### PR DESCRIPTION
**Pull request by ysantiago**

### CODE REVIEW

**Comment by ysantiago** [comment]
Runbot red is by this warning

skitch 47

Reported in #37

**Comment by ysantiago** [comment]
@moylop260 can you review this?

Thanks

File: l10n_pa_localization/model/res_city.py [hunk 1]
```diff
@@ -81,40 +81,45 @@ def _auto_init(self, cr, context=None):
...
+                "res_hood_name_country_id_state_id_district_id ON "
+                "res_country_state_district_township_hood "
+                "(name, country_id, state_id)")
 
         # index Hood name, country_id, state_id, district_id, township_id
         indexname = 'res_hood_name_country_id_state_id_district_id_township_id'
         cr.execute(
             "SELECT indexname FROM pg_indexes "
-            "WHERE indexname =\'%s\'" % (indexname))
+            "WHERE indexname IN %s", (tuple(indexname),))
```
Comment (moylop260) [inline l10n_pa_localization/model/res_city.py]: `s/IN/=/g`

File: l10n_pa_localization/model/res_city.py [hunk 2]
```diff
@@ -81,40 +81,45 @@ def _auto_init(self, cr, context=None):
...
+                "CREATE INDEX res_hood_name_country_id_state_id ON "
+                "res_country_state_district_township_hood "
+                "(name, country_id, state_id)")
 
         # index Hood name, country_id, state_id, district_id
         indexname = 'res_hood_name_country_id_state_id_district_id'
         cr.execute(
             "SELECT indexname FROM pg_indexes "
-            "WHERE indexname =\'%s\'" % (indexname))
+            "WHERE indexname IN %s", (tuple(indexname),))
```
Comment (moylop260) [inline l10n_pa_localization/model/res_city.py]: `s/IN/=/g`

File: l10n_pa_localization/model/res_city.py [hunk 3]
```diff
@@ -81,40 +81,45 @@ def _auto_init(self, cr, context=None):
...
-                '(name, country_id)' % (indexname))
+                "CREATE INDEX res_hood_name_country_id ON "
+                "res_country_state_district_township_hood (name, country_id)")
 
         # index Hood name, country_id, state_id
         indexname = 'res_hood_name_country_id_state_id'
         cr.execute(
             "SELECT indexname FROM pg_indexes "
-            "WHERE indexname =\'%s\'" % (indexname))
+            "WHERE indexname IN %s", (tuple(indexname),))
```
Comment (moylop260) [inline l10n_pa_localization/model/res_city.py]: `s/IN/=/g`

File: l10n_pa_localization/model/res_city.py [hunk 4]
```diff
@@ -81,40 +81,45 @@ def _auto_init(self, cr, context=None):
         indexname = 'res_hood_name_country_id'
         cr.execute(
             "SELECT indexname FROM pg_indexes "
-            "WHERE indexname = \'%s\'" % (indexname))
+            "WHERE indexname IN %s", (tuple(indexname),))
```
Comment (moylop260) [inline l10n_pa_localization/model/res_city.py]: `s/IN/=/g`

**Pull request merged**